atom/common/native_mate_converters/value_converter.cc

```diff
@@ -30,6 +30,27 @@ v8::Local<v8::Value> Converter<base::DictionaryValue>::ToV8(
   return converter.ToV8Value(&val, isolate->GetCurrentContext());
 }
 
+bool Converter<base::Value>::FromV8(v8::Isolate* isolate,
+                                    v8::Local<v8::Value> val,
+                                    base::Value* out) {
+  atom::V8ValueConverter converter;
+  std::unique_ptr<base::Value> value(
+      converter.FromV8Value(val, isolate->GetCurrentContext()));
+  if (value) {
+    base::Value* t = value.get();
+    std::swap(out, t);
```

Comment:
i don't think this does what you want? `std::swap` will just swap the address that the local variable `out` points to, not the actual data behind it.

Reply:
I _think_ what you'd want is

```cpp
  atom::V8ValueConverter converter;
  std::unique_ptr<base::Value> value(
    converter.FromV8Value(val, isolate->GetCurrentContext()));
  if (value) {
    *out = value->Clone();
    return true;
  }
  return false;
```